src/adapters/codemirror/features/jump_to.ts

```diff
@@ -24,8 +24,14 @@ export class JumpToDefinition extends CodeMirrorLSPFeature {
     return this.jupyterlab_components.jumper;
   }
 
-  async handle_jump(locations: lsProtocol.Location[]) {
+  async handle_jump(
+    location_or_locations: lsProtocol.Location | lsProtocol.Location[]
+  ) {
     let connection = this.connection;
+    // some language servers appear to return a single object
+    const locations = Array.isArray(location_or_locations)
+      ? location_or_locations
+      : [location_or_locations];
```

Comment:
@krassowski heh, browser testing actually found a "feature" in some of the language servers (first encountered alphabetically at `css`, so not sure how many others are covered)... I can make a separate PR of this, if you like...